From c1775b2115ee6632f41fe82ab34b1f1fabab8c52 Mon Sep 17 00:00:00 2001
From: Michal Bachraty <michal.bachraty@gmail.com>
Date: Mon, 11 Sep 2023 16:43:25 +0200
Subject: [PATCH 4/4] add more messages for cleanup_flash_dir

Signed-off-by: Michal Bachraty <michal.bachraty@gmail.com>
---
 dcs_deploy.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/dcs_deploy.py b/dcs_deploy.py
index 56aea58..59d6ae2 100755
--- a/dcs_deploy.py
+++ b/dcs_deploy.py
@@ -353,7 +353,9 @@ class DcsDeploy:
         self.prepare_status = ProcessingStatus(os.path.join(self.flash_path, "prepare_status.json"), initial_group="prepare")
     
     def cleanup_flash_dir(self):
-            cmd_exec("sudo rm -r " + self.flash_path)
+            print("cleanup_flash_dir...")
+            cmd_exec(f"sudo rm -rf {self.flash_path} && sync")
+            print("creating: " + self.flash_path)
             os.makedirs(self.flash_path)
 
     def check_dependencies(self):
-- 
2.34.1

